cdap-examples/UserProfiles/README.md

```diff
@@ -0,0 +1,72 @@
+# User Profiles Example
+
+Example application that manages user profiles in a Table. The fields of a user
+profile are updated in different ways:
+
+- Attributes like name and email address are changed through a REST
+  call when the user updates his profile;
+- The time of the last login is updated by a sign-on service every
+  time the user logs in, also through a REST call;
+- The time of the last activity is updated by a flow that processes
+  events whenever it encounters an event from that user.
+
+Features introduced: Column-Level Conflict Detection.
+
+This application uses a table with conflict detection at the column level.
```

Comment:
This application uses a table with conflict detection at the column-level. \[Shall we standardize on "row-level" and "column-level", both with hyphens?]